packages/sanity/src/structure/panes/document/documentPanel/DocumentPanel.tsx

```diff
@@ -183,7 +183,6 @@ export const DocumentPanel = function DocumentPanel(props: DocumentPanelProps) {
                   >
                     <FormView
                       hidden={formViewHidden}
-                      key={documentId + (ready ? '_ready' : '_pending')}
```

Comment:
Kind of worried about this change, is this strictly necessary? 
We could end with some inner components and states not reseting when a document changes

Reply:
Good callout, and I forgot to mention this in the PR, but this line came as a workaround for an earlier timing issue with the now removed `withDocument`-HOC.

We already have a key on document id + type in [DocumentPane](https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/structure/panes/document/DocumentPane.tsx#L133) so think this should be safe.